static/styles/settings.scss

```diff
@@ -1151,6 +1151,9 @@ input[type=checkbox].inline-block {
             visibility: visible;
             z-index: 99;
         }
+        #user-avatar-source.avatar-change-disabled {
```

Comment:
Why are you adding this selector when you can just modify the `bottom: 19%;` property on line 1121?

Reply:
It needs both those; `bottom: 19%` aligns it correctly for admins/people that can change their avatar. (guesswork) and that's because the `.user-avatar-section .avatar-controls` has a `margin-top: 20px;` which  affects the length that we compute with `bottom:19%`

Reply:
@YashRE42 Okay, so looking into the CSS more, the issue is that `#user-avatar-source` is being positioned relative to `.user-avatar-section`, which is wrong because the height of `.user-avatar-section` gets changed when `.avatar-controls` is hidden, hence the position change from the bottom.

Instead of changing the `bottom` property, depending on whether `.avatar-controls` is hidden or not, the proper solution would be to set `position: relative` for `#user-settings-avatar` instead of `.user-avatar-section` because the height of  `#user-settings-avatar` never changes. That way there is no need to modify the bottom property to keep its position static.

Can you make the necessary changes to achieve this?

Reply:
Sure, I'll do that.